corehq/messaging/smsbackends/vertex/models.py

```diff
@@ -65,7 +65,17 @@ def populate_params(self, msg_obj):
         }
         return params
 
+    def phone_number_is_valid(self, phone_number):
+        phone_number = strip_plus(phone_number)
+        # Phone number must be an Indian phone number
+        # Also avoid processing numbers that are obviously too short
+        return phone_number.startswith('91') and len(phone_number) > 3
```

Comment:
i don't think we need to be too strict here since the gateway will also reject invalid destination numbers. what this does is just catch 95% of invalid numbers so that we avoid processing them all together